docs/src/pages/[platform]/getting-started/usage/nextjs/react.mdx

```diff
@@ -61,6 +46,23 @@ Open up the project in your IDE and replace the contents of the `pages/index.js`
     <BasicShoppingCard />
   </Example>
 
+  </TabItem>
+    <TabItem title="App Router (Next.js 13.4+)">
+    <Alert variation="info" role="none">
+      Next.js 13.4+ introduced [app directory](https://nextjs.org/docs/app/building-your-application/routing#the-app-directory) with the usage of [Server Components](https://nextjs.org/docs/getting-started/react-essentials#server-components). For simplicity, in this guide, we placed `"use client"` at the top of `page.jsx` to mark it as a Client Component, but you could also wrap the UI components in your own Client Component.
```

Comment:
Thinking the below could be somewhat confusing for users:
```
For simplicity, in this guide, we placed `"use client"` at the top of `page.jsx` to mark it as a Client Component, but you could also wrap the UI components in your own Client Component.
```

Maybe we just remove it?